common/web/gesture-recognizer/README.md

```diff
@@ -0,0 +1,22 @@
+# Keyman:  Gesture Recognizer Module
+The Original Code is (C) 2022-2022 SIL International
+
+The Gesture Recognizer module is an internal component for all of Keyman's Web-based OSKs.  At this time, this is only referenced by
+KeymanWeb, seen within this repo at /web/.
+
+## Minimum Compilation Requirements
+
+* A local installation of [Node.js](https://nodejs.org/) v8.9+.
+	* Builds will call `npm install` to automatically install further necessary build dependencies.
```

Comment:
I suggest we leave this out here. `build.sh configure` should do the necessary `npm ci` or `npm install`; `build.sh build` shouldn't do any of that.